tests/common/mod.rs

```diff
@@ -23,3 +82,10 @@ pub fn pd_addrs() -> Vec<String> {
         .map(From::from)
         .collect()
 }
+
+fn enable_multi_region() -> bool {
+    match env::var(ENV_ENABLE_MULIT_REGION) {
+        Ok(s) => s == "1" || s.to_lowercase() == "true",
```

Comment:
I think it is usual practice to enable a feature if the env var is set to anything

Reply:
That's my first thought too. Then I thought `MULTI_REGION=0` would be weird. I don't know a lot about the convention here.

Reply:
I think you must unset an env var not set it to 0 if you want it to be falsey

Reply:
OK